docs/microbit_micropython_api.rst

```diff
@@ -266,6 +320,25 @@ There is an I2C bus on the micro:bit that is exposed via the `i2c` object.  It h
     # write buf to device with addr; repeat=True means a stop bit won't be sent.
     i2c.write(addr, buf, repeat=False)
 
+Sounds **V2**
+------
+
+A set of expressive sounds are available to the micro:bit **V2** and can be
+accessed via the ``audio`` module.
+
+**Built-in sounds**
+
+``Sound.GIGGLE``
+``Sound.HAPPY``
+``Sound.HELLO``
+``Sound.MYSTERIOUS``
+``Sound.SAD``
+``Sound.SLIDE``
+``Sound.SOARING``
+``Sound.SPRING``
+``Sound.TWINKLE``
+``Sound.YAWN``
```

Comment:
Was this how you expected the rendering?
![image](https://user-images.githubusercontent.com/29712657/100260346-3431f100-2f41-11eb-8923-7f31bde8d5bf.png)

Reply:
Yes, it's similar to how gestures are listed. But we could make it an unordered list.

Reply:
No no, that's fine, is just that the other example I saw was Image and that was an item per line in a big block. To be fair I think this is just as readable and takes less space.
We could do the same for Image, that code block is 3 times larger than anything else.

Reply:
Have updated the image list in https://github.com/bbcmicrobit/micropython/pull/692/commits/cff8eee60dca43be87266cb8a00545a1ef90ee00